<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionPostalWindow</class>
 <widget class="QDialog" name="ConnectionPostalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mail Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset theme="internet-mail">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="KTitleWidget" name="protocolTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Shows how and where you're communicating.</string>
   </property>
   <property name="text">
    <string>Untitled Mail Protocol</string>
   </property>
   <property name="icon">
    <iconset theme="internet-mail">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>1031</width>
     <height>581</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="readTab">
    <attribute name="icon">
     <iconset theme="internet-mail">
      <normaloff>.</normaloff>.</iconset>
    </attribute>
    <attribute name="title">
     <string>Read</string>
    </attribute>
    <widget class="QTreeWidget" name="foldersView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>256</width>
       <height>511</height>
      </rect>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QListWidget" name="messagesView">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>731</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="composeTab">
    <attribute name="icon">
     <iconset theme="edit">
      <normaloff>.</normaloff>.</iconset>
    </attribute>
    <attribute name="title">
     <string>Compose</string>
    </attribute>
    <widget class="KLineEdit" name="recipientField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>661</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhEmailCharactersOnly</set>
     </property>
     <property name="placeholderText">
      <string>Recipient Email Address...</string>
     </property>
    </widget>
    <widget class="KLineEdit" name="ccField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>1001</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhEmailCharactersOnly</set>
     </property>
     <property name="placeholderText">
      <string>CC Email Address...</string>
     </property>
    </widget>
    <widget class="KLineEdit" name="bccField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>1001</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhEmailCharactersOnly</set>
     </property>
     <property name="placeholderText">
      <string>BCC Email Address...</string>
     </property>
    </widget>
    <widget class="KLineEdit" name="subjectField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>1001</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="placeholderText">
      <string>Subject...</string>
     </property>
     <property name="clickMessage">
      <string>Subject...</string>
     </property>
    </widget>
    <widget class="KRichTextEdit" name="messageField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>1001</width>
       <height>221</height>
      </rect>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoAll</set>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="basicToolbar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QToolButton" name="italicButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Italic</string>
        </property>
        <property name="icon">
         <iconset theme="format-text-italic">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="boldButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bold</string>
        </property>
        <property name="icon">
         <iconset theme="format-text-bold">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="underlineButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Underline</string>
        </property>
        <property name="icon">
         <iconset theme="format-text-underline">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="alignLeft">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Align To Left</string>
        </property>
        <property name="icon">
         <iconset theme="format-justify-left">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="alignCenter">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Align To Center</string>
        </property>
        <property name="icon">
         <iconset theme="format-justify-center">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="alignRight">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Align To Right</string>
        </property>
        <property name="icon">
         <iconset theme="format-justify-right">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="justifyButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Justify</string>
        </property>
        <property name="icon">
         <iconset theme="format-justify-fill">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KColorButton" name="textColor">
        <property name="text">
         <string>Text Color</string>
        </property>
        <property name="icon">
         <iconset theme="com.github.cassidyjames.palette">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="sendBar">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>460</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QToolButton" name="sendButton">
        <property name="text">
         <string>Send</string>
        </property>
        <property name="icon">
         <iconset theme="mail-send">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="discardButton">
        <property name="text">
         <string>Discard</string>
        </property>
        <property name="icon">
         <iconset theme="org.kde.plasma.trash">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="KLineEdit" name="senderNameField">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>30</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="placeholderText">
      <string>Your Name...</string>
     </property>
     <property name="clickMessage">
      <string>Your Name...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>KTextEdit</extends>
   <header>krichtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
